### PR TITLE
Added make module-init command in Readme

### PR DESCRIPTION
**Pull request by dipankardas011**
A command inside Readme.md was missing for starting of local docker container

Signed-off-by: Dipankar Das <dipankardas0115@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ git submodule update --init --recursive --depth 1
 To build the site in a container, run the following:
 
 ```bash
+make module-init
```
Comment (sftim) [inline README.md:42]:
```suggestion
# if you didn't yet run "git submodule update --init --recursive --depth 1",
# then this next command does that for you. You don't need to do this
# each time.
make module-init

```

People don't need to to this every time, and they only need to do that if they didn't follow the advice in the section immediately before this one.

**Review by sftim — COMMENTED** [review]
How about this compromise?

File: README.md (hunk 1 above)
Comment (dipankardas011) [reply README.md:42]: Understood will change it👍

**Comment by dipankardas011** [comment]
If the Git command in prerequisite section could be replaced with make command it would be much better
As the first time contributors don't need to know how dependencies are fetched just make and done!
Would like to see your thoughts

**Review by sftim — COMMENTED** [review]
/lgtm

We could make the instructions better for Windows folks, and this makes it a bit worse. However as the advice for Windows-using contributors is already not great, I'm OK with that.

**Comment by dipankardas011** [comment]
> /lgtm
> 
> We could make the instructions better for Windows folks, and this makes it a bit worse. However as the advice for Windows-using contributors is already not great, I'm OK with that.

I van add both the instructions i.e
For Windows Git command
For linux make command

File: README.md [hunk 2]
```diff
@@ -29,11 +29,18 @@ cd website
 
 The Kubernetes website uses the [Docsy Hugo theme](https://github.com/google/docsy#readme). Even if you plan to run the website in a container, we strongly recommend pulling in the submodule and other development dependencies by running the following:
 
-```bash
+### Windows
+```powershell
 # pull in the Docsy submodule
```
Comment (sftim) [inline README.md:34]:
```suggestion
# fetch submodule dependencies
```

File: README.md [hunk 3]
```diff
@@ -29,11 +29,18 @@ cd website
...
 git submodule update --init --recursive --depth 1
 ```
 
+### Linux
+```bash
+# fetch all the dependencies
```
Comment (sftim) [inline README.md:40]:
```suggestion
# fetch submodule dependencies
```

File: README.md [hunk 4]
```diff
@@ -29,11 +29,18 @@ cd website
...
 The Kubernetes website uses the [Docsy Hugo theme](https://github.com/google/docsy#readme). Even if you plan to run the website in a container, we strongly recommend pulling in the submodule and other development dependencies by running the following:
 
-```bash
+### Windows
+```powershell
 # pull in the Docsy submodule
 git submodule update --init --recursive --depth 1
 ```
 
+### Linux
```
Comment (sftim) [inline README.md:38]: Optionally:
```suggestion
### Linux / other Unix
```

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

**Comment by dipankardas011** [comment]
Ig now it's mergeable

**Comment by dipankardas011** [comment]
Squashed the Git commits

**Comment by dipankardas011** [comment]
@sftim don't know why again Netlify tests failed 😄

**Comment by sftim** [comment]
> could you check this PR are there any pending reviews left?

As with all PRs, this PR can only merge if it gets LGTM _and_ approval. That hasn't happened yet. I'm happy for any website maintainer to do the next review on this PR.

**Comment by tengqm** [comment]
@dipankardas011 You may want to recommit this PR to trigger another gate test.

**Comment by dipankardas011** [comment]
@tengqm Done!

**Comment by tengqm** [comment]
Re-applying lgtm and approval.
/lgtm
/approve

**Pull request merged**